app/helpers/flash_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FlashHelper
   # Flash type -> Bootstrap alert class
   def flash_mapping
-    { notice: :info, error: :danger, success: :success }
+    { notice: :info, error: :danger, authorization_error: :danger, success: :success }
```

Comment:
What @sbrnunes said :) This new flash type should be specific to only those cases where we want to display the access request link, I don't believe this should be generic functionality.

I'm not sure that the stackoverflow solution improves the situation much, however you want to proceed, in the end you still need to add a new flash type.